archive/p/powershell/README.md

```diff
@@ -14,4 +15,7 @@ Welcome to Sample Programs in PowerShell!
 ## References
 
 - [PowerShell Documentation](https://docs.microsoft.com/en-us/powershell/)
-- [Try It Online: PowerShell](https://tio.run/#powershell)
\ No newline at end of file
+- [Try It Online: PowerShell](https://tio.run/#powershell)
+
+[1]: .\hello-world.ps1
```

Comment:
Is the intent to be able to run the scripts with these links? That's actually pretty cool! But if a user is browsing GitHub, the links don't work.

At any rate, I usually add links later when my articles are ready.

Reply:
I actually put the links in as a force of habit since that's how my local machine is set up. I'll update this accordingly and push a new commit.